src/components/Node.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <q-card id="node" class="shadow-1" :style="style">
     <q-card-section class="header non-selectable text-center" v-touch-pan.mouse="onPan">
-      <img id="icon" class="non-selectable" v-if="instance.spec.icon" :src="instance.spec.icon" alt="">
+      <img id="icon" class="non-selectable" v-if="instance.spec.icon" :src="instance.spec.icon" alt />
```

Comment:
Might the `alt` prop be safe to delete here? Is not setting it equivalent to setting it empty?

Reply:
My VS Code plugin apparently thinks so, yes.

Reply:
We likely do not require an alternate text for the icons, but that is not the point of this branch.